aries-site/src/examples/templates/dashboards/components/UpdatesFeed.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import { Card, CardBody, CardHeader, ThemeContext } from 'grommet';
+import { useContext } from 'react';
+import { DashboardCardHeader } from '.';
+
+export const UpdatesFeed = ({ background, children, title }) => {
+  const theme = useContext(ThemeContext);
+  const { body, header } = theme.card;
+
+  return (
+    <Card background={background || 'background'}>
+      <CardHeader
+        pad={{ horizontal: header.pad, top: header.pad, bottom: 'none' }}
+      >
+        <DashboardCardHeader
+          menuItems={[
+            { label: 'Move', onClick: () => {} },
+            { label: 'Share', onClick: () => {} },
+          ]}
```

Comment:
Since this is a reusable component, the menuItems value should be passed through a prop. That way the caller can choose what the menuItems are.

```suggestion
          menuItems={menuItems}
```